Example/Example for Delirium/Modules/StickyCollectionViewLayout/StickyCollectionViewController.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+import UIKit
+import Delirium
+
+enum StickyCellIdentifier: String {
+    case stickyCell
+}
+
+enum Row: Int {
+    case row1, row2, row3, row4, row5, row6, row7, row8, row9, row10, row11, row12, row13, row14, row15, row16, row17, row18
+}
+
+enum Column: Int {
+    case A, B, C, D, E, F, G, H, I, J, K, L, M, N, O, P, Q, R, S, T, U, V, W, X, Y, Z, last
+}
+
+class ViewModel {
+    private var model: [Row: [Column]]
+
+    init(_ model: [Row: [Column]]) {
+        self.model = model
+    }
+
+    var numberOfSection: Int {
+        return model.count
+    }
+
+    func rows(in row: Row) -> Int {
+        guard let rows = model[row] else {
+            return 0
+        }
+        return rows.count
+    }
+
+    func row(in row: Row) -> [Column] {
+        return model[row] ?? [Column]()
+    }
+
+}
+
+class StickyCollectionViewController: UIViewController {
+
+    @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var stickyCollectionViewLayout: StickyCollectionViewLayout!
+
+    var viewModel: ViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let row1: [Column] = [.A, .B, .C, .D, .E, .F, .G, .H, .last]
```

Comment:
I would move this to the viewModel instead of setting the data here. This is viewModel functionality.